src/Weather/Skysight/SkysightAPI.hpp

```diff
@@ -47,6 +47,7 @@ struct BrokenDateTime;
 class SkysightAPI final: public Timer {
   friend struct SkysightRequest;
   friend struct SkysightAsyncRequest;
+  friend class  CDFDecoder;
```

Comment:
Two spaces? Don't draw ASCII art in code.

Reply:
remove a space.